File: BookStore.BuisneLogic/Models/PrintingEditions/PrintingEditionModelItem.cs
using BookStore.BusinessLogic.Models.Authors;
using BookStore.BusinessLogic.Models.Base;
using static BookStore.BusinessLogic.Models.Enums.Enums;

namespace BookStore.BusinessLogic.Models.PrintingEditions
{
    public class PrintingEditionModelItem : BaseModel
    {
        public long Id { get; set; }
        public string Title { get; set; }
        public string Description { get; set; }
        public decimal Price { get; set; }
        public TypePrintingEdition ProductType { get; set; }
        public AuthorModel Authors { get; set; }
    }
}

File: StoreClient/src/app/shared/models/PeintingEditions/printing-edition-model-item.ts
import {AuthorModel} from '../Authors/author-model';
import { TypePrintingEdition } from 'src/app/shared/enums/type-printing-edition';
import { Currencys } from '../../enums/currencys';
import { BaseModel } from '../Base/base-model';

export class PrintingEditionModelItem extends BaseModel {
        id: number;
        title: string;
        description: string;
        price: number;
        productType: TypePrintingEdition;
        currencys: Currencys;
        authors: AuthorModel;
}
